pkg/report/sarif.go

```diff
@@ -0,0 +1,224 @@
+package report
+
+import (
+	"fmt"
+	"html"
+	"io"
+	"regexp"
+	"strings"
+
+	"github.com/owenrumney/go-sarif/v2/sarif"
+	"golang.org/x/xerrors"
+
+	"github.com/aquasecurity/fanal/analyzer/os"
+	"github.com/aquasecurity/fanal/types"
+)
+
+const (
+	sarifOsPackageVulnerability        = "OsPackageVulnerability"
+	sarifLanguageSpecificVulnerability = "LanguageSpecificPackageVulnerability"
+	sarifConfigFiles                   = "ConfigFileMisconfiguration"
```

Comment:
It should be descriptive enough.

```suggestion
	sarifConfigFiles                   = "Misconfiguration"
```